tests/backends/base/test_base_async.py

```diff
@@ -0,0 +1,14 @@
+import unittest
+
+from django.db import connection, new_connection
+from django.test import SimpleTestCase
+
+
+class AsyncDatabaseWrapperTests(SimpleTestCase):
+    @unittest.skipUnless(connection.supports_async is True, "Async DB test")
+    async def test_async_cursor(self):
```

Comment:
This fails on postgres with connection pools (`"OPTIONS": {"pool": True}`):
`AttributeError: 'coroutine' object has no attribute 'set_autocommit'`

Reply:
fixed in 9ddddab18b7fe155bfd7fb99f897fbaf2ecce6a6